### DISCUSSION TITLE
nested swiper with thumbs

### DISCUSSION THREAD

**Discussion by quartopianodesign** [nolimits4web/swiper#5730] [Q&A] [answered]
Hi everyone,
I'm trying to implement a nested swiper with thumbs gallery.
I can't trigger the thumb click.
Anyone have any suggestions?
Thanks!

https://codesandbox.io/s/test-nested-swiper-with-thumbs-hvxh78

**Comment by Arhell** [comment, marked as answer]
Try this option 
https://codesandbox.io/s/ldhjv3?file=/index.html
https://swiperjs.com/demos#thumbs-gallery-loop

**Reply by quartopianodesign** [reply]
The option is already set, but mine is a nested slider ...
